package/MDAnalysis/analysis/density.py

```diff
@@ -501,6 +501,7 @@ def current_coordinates():
     pm = ProgressMeter(u.trajectory.n_frames, interval=interval, quiet=quiet,
                        format="Histogramming %(n_atoms)6d atoms in frame "
                        "%(step)5d/%(numsteps)d  [%(percentage)5.1f%%]\r")
+    start, stop, step = u.trajectory.check_slice_indices(start, stop, step)                    
```

Comment:
Yeah using None will default them to those values.  The slice `a[::10]` is identical to `a[None:None:10]` in Python